src/Diagnostics.ModelsAndUtils/Diagnostics.ModelsAndUtils.csproj

```diff
@@ -18,7 +18,7 @@
   
   <ItemGroup>
     <PackageReference Include="CommonMark.NET" Version="0.15.1" />
-    <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.2" />
```

Comment:
@ShekharGupta1988 do you foresee a problem here?

Reply:
@dagrooms52 do we need to bump up the Newtonsoft version?

Reply:
Try building the solution and verify that:
a) There are no warnings related to newtonsoft assembly version mismatch between our projects
b) in the runtimehost build output, newtonsoft dll is present (it should)
c) latest version is there.

@hforeste - In source depot, the problem was there were many dependent projects using different versions of newton soft which use to create problem.

In our case, if a) and b) are there, then it should be perfectly fine.

Reply:
This was just a consolidation step, so the 12.0.2 dll should be present as it was before.